### ISSUE TITLE
Images are truncated? 

### ISSUE THREAD

**Issue by igrigorik** [1000ch/webponize#4]
When I drag in a ~8MB JPEG, it spits out a ~50KB webp.. which is a valid file, but it looks like it's only encoding a tiny fragment of the image (top right corner). Known issue?

**Comment by 1000ch** [comment]
I have encountered, too.

But... after saving it (broken? JPEG) with Photoshop, It was successfully converted. So would you try to re-save (to fix?) the big JPEG and convert it?

Should be inspected anyway :(

**Comment by igrigorik** [comment]
Nope, same issue. 

![image](https://cloud.githubusercontent.com/assets/10652/7965647/b54ee6f6-09d3-11e5-99e7-e79b551e8a9a.png)

**Comment by 1000ch** [comment]
Umm, I will fix it soon. Thank you for reporting.

**Comment by 1000ch** [comment]
```c
// Returns the size of the compressed data (pointed to by *output), or 0 if
// an error occurred. The compressed data must be released by the caller
// using the call 'free(*output)'.
// These functions compress using the lossy format, and the quality_factor
// can go from 0 (smaller output, lower quality) to 100 (best quality,
// larger output).
WEBP_EXTERN(size_t) WebPEncodeRGB(const uint8_t* rgb,
                                  int width, int height, int stride,
                                  float quality_factor, uint8_t** output);
```

https://github.com/1000ch/webponize/blob/master/WebPonize/libwebp/encode.h#L43-51

This does not return 0, so the issue might be in `libwebp.a`

**Comment by 1000ch** [comment]
@igrigorik This is fixed on `v1.1.6`.
This was the problem of my code ;(
